umbral/umbral.py

```diff
@@ -64,6 +122,27 @@ def __init__(self, point_eph_e, point_eph_v, bn_sig):
 
         self.cfrags = {}
 
+    @classmethod
+    def from_bytes(self, data: bytes, curve):
+        """
+        Instantiates a Capsule object from the serialized data.
+        """
+        eph_e = Point.from_bytes(data[0:33], curve)
+        eph_v = Point.from_bytes(data[33:66], curve)
+        sig = BigNum.from_bytes(data[66:98], curve)
+
+        return Capsule(eph_e, eph_v, sig)
```

Comment:
Again, if this is truly a classmethod, logically speaking, let's use the class object instead of the module-level namespace here.

Or, if you want a staticmethod instead, just do that.

Reply:
You mean, using `cls(eph_e, ...)` if we use `classmethod`?

Reply:
Yeah, exactly.